js/server.js

```diff
@@ -38,7 +38,7 @@ var Server = function(config, callback) {
 			res.status(403).send("This device is not allowed to access your mirror. <br> Please check your config.js or config.js.sample to change this.");
 		});
 	});
-	app.use(helmet());
+	app.use(helmet(config.helmetOptions));
```

Comment:
I agree with @roramirez. I thing the default situation should be no change to the current configuration.